lib/ClientManager.test.js

```diff
@@ -50,10 +50,10 @@ describe('ClientManager', () => {
         // should still have client - grace period has not expired
         assert(manager.hasClient('foobar'));
 
-        // wait past grace period (1s)
-        await new Promise(resolve => setTimeout(resolve, 1500));
+        // wait past grace period
+        await new Promise(resolve => setTimeout(resolve, 10001));
```

Comment:
Since this is a fork of https://github.com/StyleT/mytunnel-server I'd like to not grow the diff a lot. 
Our version only differs by hardcoded timeouts. 
I think it is a great idea to ask maintainer if he thinks that making timeouts configurable makes sense, and if so send pull request and stop maintaining own fork.